packages/nextjs/hooks/scaffold-eth/useScaffoldEventHistory.ts

```diff
@@ -49,74 +50,82 @@ export const useScaffoldEventHistory = <
 
   useEffect(() => {
     async function readEvents() {
-      if (!contract) {
-        console.log("contract not found");
-        return;
-      }
       try {
-        const fragment = contract.interface.getEvent(eventName);
-        const emptyIface = new ethers.utils.Interface([])
-        const topicHash = emptyIface.getEventTopic(fragment)
-        const topics = <any>[topicHash];
+        if (!deployedContractData || !contract) {
+          console.error("Contract not found");
+          setEvents([]);
```

Comment:
In relation to my other comment, here we can then do `setEvents(undefined)` to make the difference between "no info could be determined" and "we found the results, but it's empty"

Reply:
Changed!